### ISSUE TITLE
Windows beep sound feedback on first pressing CapsLock (NVDA modifier key) when Toggle keys (EAC) are enabled

### ISSUE THREAD

**Issue by DrSooom** [nvaccess/nvda#9277]
### Path:
NVDA menu » Preferences » Settings... » Keyboard

Quote from [my comment on issue #9274](https://github.com/nvaccess/nvda/issues/9274#issuecomment-463478544):
> ### Steps to reproduce:
> 1. Enable "Toggle keys" via the EAC/Keyboard settings (WIN+U) on Windows 7.
> 2. Set the CapsLock key as a NVDA modifier key, which is the default behaviour on all my machines – and which is recommended for notebook/laptop users too.
> 3. And now press the CapsLock key twice.
> ### Actual behaviour:
> 1. Every time when the CapsLock key is pressed, a beep sound appears. So if you press it twice, two beeps appear. And every time when you are pressing NVDA+[doesn't matter], it beeps.
> 2. And the beep sound frequency always reports the previous status of this key.
>   + So if CapsLock is turned on, the lower tone appears.
>   + And if CapsLock is turned off, the higher tone appears.
> 3. Normally the higher tone appears after the key was switched on and the lower after it was switched off.

### Expected behaviour:
The sound feedback should be disabled on the first press of the CapsLock key by the user via a simple checkbox in the Keyboard NVDA Settings. @leonardder prefer this beep sound feedback based on [his comment on issue #9274](https://github.com/nvaccess/nvda/issues/9274#issuecomment-463515158) and I absolutely hate it. So the user should have the possibility to change this behaviour. I can't say which behaviour should be the default one.
#### NVDA installed/portable:
Both
#### NVDA version:
2018.4.1
#### Windows version:
Win7 (tested) to Win10-1903

**Comment by leonardder** [comment]
While I acknowledge this issue, I'm afraid there's nothing we could do about this. Even while NVDA hooks the keyboard at the lowest level it could, it does not stop the toggle key beeps from playing. Another example of this is controlling a machine using NVDA Remote while toggle keys are on. You will notice that the toggle key beeps will still play, yet none of the keyboard keys are processed by the system itself.

Closing as can't fix
